src/client/containers/SlackAuthPage/SlackAuthPage.component.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+
+const axios = require('axios');
+
+export default function SlackAuthPage() {
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
+    const code = params.get('code');
+    axios.post(`/api/signin?code=${code}`);
+    // .then((res) => {console.log('this is response', res)});
```

Comment:
You can remove this part.
Also it would be nice to `try{}catch{}` when doing e.g. an API call, in case it fails